src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1450,4 +1451,25 @@ else if (item instanceof ItemMonsterPlacer)
         }
         return modId;
     }
+
+    /**
+     * Posts the world unload event.
+     * Optionally catches and suppresses any exceptions to allow the game to safely finish unloading the world.
+     */
+    public static boolean onWorldUnload(World world, boolean suppressExceptions)
+    {
+        if (!suppressExceptions)
+        {
+            return MinecraftForge.EVENT_BUS.post(new WorldEvent.Unload(world));
```

Comment:
This is a bit redundant, perhaps
```java
if (!suppressExceptions) throw e;
```
in the catch block would be better?

Reply:
I didn't think I could throw a `Throwable` without wrapping it first, so I avoided it. I'll fix this.